wptechprodigy/react-todo
# Added Checking and Deleting completed todoitem functionality

#### What does this PR do?
Adds checking and deleting completed todos
#### Description of what task was completed
Added functionality to be able to check completed todos <br>
When a todo is checked, it's stricken through.
The ability to delete the completed todos is added as well.
#### How can this functionality be tested?
1. Clone the project repo.
2. From the root folder run `npm run start` or simply `npm start` .
3. Check the checkbox to the left of each hard-coded todo to test the strike-through feature
4. Click of the `X` to the right of any todo to delete it.
